src/docs/sdk/performance/modules/assets.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Assets Module'
+---
+
+The javascript SDK should automatically instrument most assets fetched by the application. This includes images, stylesheets, scripts, and other assets. But if you may wish to instrument assets manually.
```

Comment:
This documentation is aimed at SDK developers, so you don't need to mention the "javascript SDK" specifically, nor talk about manual instrumentation, since this is a different usecase. On the flipside, you can (and should) be specific about what kinds of assets to instrument! e.g., this is a good time to point out that `link` (right?) elements are not supported and so on

Reply:
Although to your point, it probably makes sense to explain that resources only make sense in a Browser context!

Reply:
Ohh, i actually didn't know this was for SDK developers, that makes a lot more sense!